### DISCUSSION TITLE
GitHub Education

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#38301] [GitHub Education]
What is GitHub Education?

**Comment by david-engelmann** [comment]
It's benefits offered to students! Basically, premium features If you can verify you are a student! [Check it out!](https://education.github.com/)

**Comment by SilasRodrigues19** [comment]
@iLakshya GitHub Education helps students, teachers and schools access the best tools that are provided through GitHub partners, learning and training events. You can see more about the GitHub Education Programs in this link: [https://education.github.com/](https://education.github.com/)
